src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -0,0 +1,160 @@
+
+Polymer [supports evergreen browsers](https://www.polymer-project.org/1.0/docs/browsers) and when combined with webcomponents.js supports IE 11+.
+
+# A quick Polymer example
+
+
+    <dom-module id="my-element">
+      <template>
+        <style>
+        .box{
+          color: grey;
+        }
+        </style>
+        <div class="box">{{mybinding}}</div>
+        </template>
+        <script>
+        // element registration
+        Polymer({
+          is: "my-element",
+
+          properties: {
+            mybinding: {
+              type: String,
+              value: "My text here"
+            }
+          }
+          });
+        </script>
+    </dom-module>
+
+The example above creates a new component <code>my-element</code> and it has one binding called <code>mybinding</code>. This can be changed programatically or with attribute binding:
+
+    <my-element id="obj" mybinding="New text"></my-element>
+
+    this.$.obj.set('mybinding','Even newer text');
+    //this.$ is a Polymer representation of local DOM nodes
+
+# PRPL pattern
+
+PRPL (pronounced purple) is a set of concepts to enable the next generation of web applications:
+
+* Push - send critical components for apps initial view
+* Render - load the initial screen ASAP
+* Pre-cache - store the elements for remaining views
+* Lazy-load - when a new view is called, load the elements
+
+This pattern relies on [HTTP 2](https://http2.github.io) for the push capability (when you request only one element or page, the server recommends other files to cache to prevent having the browser work this out at a later time). The pre-cache relies on [service workers](http://www.w3.org/TR/service-workers/) to pull down (but not instantiate) all the other elements required for the routes. These elements will then be instantiated at the point in time they are needed (and all the code for this is in the browser).
+
+# Speeding up with vulcanisation
+
+As most browsers doesn't natively support HTML imports out-of-the-box (and HTTP 2 is not widely run on severs) they need to be polyfilled. The problem is polyfills are slower than native implementations, and worse you can import an element, that imports dependent elements, and that itself imports elements.
+
+This dependency chain can be slow to import so a compiler ([vulcanise](https://github.com/Polymer/vulcanize)) was written to combine all dependencies into one file so only one HTML import is needed.
+
+The PRPL pattern is the opposite of vulcanistion. This pattern only loads the minimum individual elements to show whats on screen, then lazily load the others.
+
+Polymer's new [CLI](https://github.com/Polymer/polymer-cli) has as tooling for generating both the PRPL and vulcanised version of your project.
```

Comment:
I see now the correlation with the previous section. Consider stetting this information up earlier in the previous paragraph so that we can get a better picture of where the content is driving towards